book/python/using-connectors.md

```diff
@@ -0,0 +1,193 @@
+# Using Connectors As Sources & Sinks
```

Comment:
I think it would be great to call out for feedback right here at the beginning. This is a preview release, we want to hear from you. You can give us feedback by (mailing list, irc, issue, whatever folks think is best)